plugins/aws/resoto_plugin_aws/aws_client.py

```
@@ -146,6 +146,20 @@ def call(
             log.debug(f"The Aws endpoint does not exist in this region. Skipping. {e}")
             return None
 
+    @log_runtime
+    def call(
+        self,
+        aws_service: str,
+        action: str,
+        result_name: Optional[str],
+        expected_errors: Optional[List[str]] = None,
+        **kwargs: Any,
+    ) -> JsonElement:
+        try:
+            return self.call_single(aws_service, action, result_name, max_attempts=1, **kwargs)
+        except ClientError:
+            return None
```

why would we swallow client errors?
I would opt to remove the try/except block